assets/templates/99_master-api-haproxy.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  generation: 1
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 00-master-api-haproxy
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    systemd:
+      units:
+        - contents: |
+            [Unit]
+            Description=Load Balance OpenShift API using HAProxy
+            Wants=network-online.target
+            After=network-online.target
+
+            [Service]
+            WorkingDirectory=/etc/haproxy
+            ExecStartPre=/usr/local/bin/haproxy.sh
```

Comment:
Do we need this line?

Reply:
```suggestion
```

This causes
`Mar 20 12:29:33 master-0 haproxy.sh[3684]: Must provide parameter (start/stop)`

Reply:
Done. Thanks